lib/elixir_console_web/live/console_live/history_component.html.leex

```diff
@@ -1,5 +1,9 @@
 <div class="flex-1"></div>
 <div class="p-2" id="commandOutput" phx-update="append">
+  <div id="version-info">
+    <p>Elixir version: <%=System.version()%></p>
+    <p>Erlang version: <%=System.otp_release()%></p>
```

Comment:
nit: Adding a space after `<%=` and before `%>` helps with readability
```elixir
<p>Elixir version: <%= System.version() %></p>
<p>Erlang version: <%= System.otp_release() %></p>
```

Reply:
You are right. I thought that running mix format it might have been automatically done. Respecting the general comment on the review, I also thought on the second place-display option that you mentioned, because after a few commands the visitor stops seeing the versioning info unless the command output section is scrolled. Despite this fact, I found it the best place to put it, but I listen to your opinions on this and I change/leave it.

Reply:
Following the answer to the general comment, what if we show only one line like
`Elixir 1.11.4/OTP 24` ? Looks more consolish 🙃